### ISSUE TITLE
Set scripts source using an absolute path

### ISSUE THREAD

**Issue by houdayec** [frencojobs/derry#42]
First of all, great job on this package. It is a real time saver, I've always asked myself _"Should I create a dedicated bash script to run my commands? Nah, no time"_. You did a lot better.

However, I am currently working on multiple projects and I would like to have my `derry.yaml` common to those projects (commands are common ones). I would like to not have to create a `.yaml` file for every project, however, it seems that the `scripts` tag in `pubspec.yaml` only looks if the file exists at the root of the project. 

Would it be possible to set the source as an absolute path so it can look outside of the project?

**Comment by houdayec** [comment]
I have posted this message before trying properly, I paid too much confidence to the output of `derry source` command.

In my project1 pubspec, I have set

`scripts: ../../scripts.yaml`

Then I ran `derry source`, output:

`/Users/houdayec/flutterdev/root/projects/project1/../../scripts.yaml`

which is wrong, it should be 

`/Users/houdayec/flutterdev/root/scripts.yaml`

as it detects properly the file (otherwise I get `File Not Found `error)

**Comment by frencojobs** [comment]
The path returned by `derry source` wasn't correctly normalized. It is now fixed in `v1.2.1`.
```bash
derry source # ../../scripts.yaml
derry source -a # /home/frenco/work/tmp/scripts.yaml
# or 
# derry source --absolute
```
Thanks for taking the time to open this issue.

> First of all, great job on this package. It is a real time saver, I've always asked myself "Should I create a dedicated bash script to run my commands? Nah, no time". You did a lot better.

Thanks for the compliment too, I really appreciated it!

**Comment by houdayec** [comment]
@frencojobs Thanks for your reactivity! Keep going with your packages :)
